cmd/mayactl/app/command/stats_helper.go

```diff
@@ -4,13 +4,15 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
-	"os"
 	"time"
 
+	"github.com/openebs/maya/pkg/client/mapiserver"
+
 	"github.com/openebs/maya/pkg/util"
 	"github.com/openebs/maya/types/v1"
 )
 
+// Client interface contans GetVolAnnotation
```

Comment:
Can comment be changed to "Client interface defines the GetVolAnnotation method which can be used to get the details about the volumes from maya-apiserver." ?

Reply:
ok sure